jenkins/run-server-func-tests

```diff
@@ -73,30 +88,25 @@ if [[ ${exit_status} -ne 0 ]]; then
     podman exec ${PB_POD_NAME}-pbenchserver journalctl
     printf -- "\n--- journalctl dump ---\n\n"
 fi
-if [[ -z ${1} ]]; then
-    echo "No clean up requested ... Pbench Server container and support services pod likely still running!"
+
+if [[ -z "${cleanup_flag}" ]]; then
+    echo "No clean up requested -- the Pbench Server container and support services pod are running!"
+    trap "" EXIT
     exit ${exit_status}
 fi
```

Comment:
Is this really necessary since you're trapping `EXIT`??? Granted, cleanup may fail after this point because we might have already stopped/removed things, but that's equally true at line 49 if we failed to start them; and doing the prunes and cleanups still has value.

Also, I think the proper way to cancel a trap is `trap EXIT` or `trap - EXIT` rather than assigning a null command:

```
(.venv39) dbutenho 14:25 dashboard:antiunpack trap "" EXIT
(.venv39) dbutenho 14:26 dashboard:antiunpack trap -p
trap -- '' EXIT
(.venv39) dbutenho 14:26 dashboard:antiunpack trap EXIT
(.venv39) dbutenho 14:26 dashboard:antiunpack trap -p
```

Reply:
> Is this really necessary since you're trapping `EXIT`???

If `--cleanup` is _not_ specified, then we want to exit at  line 95 and leave everything running (or in whatever state it's currently in...); so, we need the `if` statement here (to skip the subsequent `stop` commands), and we need to remove the `trap` on `EXIT` so that the cleanup handler does _not_ get called; and, as long as we're here, we'll take the opportunity to remind the user that stuff is still running (and we do it somewhat gently, as we presume that the user omitted `--cleanup` on purpose...).

So, there is no "after this point" if there was no `--cleanup` specified, and we don't want to do the `stop`'s, `rm`'s, and `prune`'s unless the user requested them (although, I believe that the default here is a bit less than entirely user-friendly).

Nevertheless, after reading the man page for the fifth time, I concur with your assessment regarding removing `trap` handlers, and I shall fix the code.